spec/mailers/previews/ticket_preview.rb

```diff
@@ -5,4 +5,9 @@ def new_ticket
     ticket = Ticket.last
     TicketMailer.new_ticket(user:, ticket_id: ticket.id)
   end
+
+  def missing
```

Comment:
This naming is kind of confusing to me.

Missing what? You mark something as missing? You notify something is missing?

Same in `app/mailers/ticket_mailer.rb`

Reply:
I'm assuming the method told me something was missing, and the context (the ticket file) tells me what kind of thing it is. but, I will improve that method name

Reply:
Done @AgostinaDev